cmd/gpu-feature-discovery/main.go

```diff
@@ -86,6 +86,12 @@ func main() {
 			Value:   "/etc/kubernetes/node-feature-discovery/features.d/gfd",
 			EnvVars: []string{"GFD_OUTPUT_FILE"},
 		},
+		&cli.StringFlag{
+			Name:    "imex-nodes-config",
+			Usage:   "the path to nvidia-imex nodes config file",
+			Value:   "/etc/nvidia-imex/nodes_config.cfg",
```

Comment:
Why do we set a default value here? This means that a user could mount this file in and trigger the IMEX behaviour without explicitly setting the value.

Reply:
We no longer set a default value in this flag

Reply:
The code says otherwise. The `Value` property sets the default value of the flag. Should this be dropped?

Reply:
I think we need a default here. We just check for existence /  emptiness as conditions to skip generating the IMEX related labels.